models/ideas.js

```diff
@@ -19,10 +19,18 @@ function createIdea(header, description, askedSum, userId) {
 }
 
 function updateIdea(ideaId, key, value) {
+    if (!value) {
+        throw new Error(`Error. You didn't provide any value to update`);
+    }
+
     let ideas = getIdeas();
 
     const idea = ideas.find((idea) => ideaId === idea.id);
 
+    if (!idea) {
+        throw new Error(`Error. No idea with such ID found..`);
```

Comment:
Do you want to mention which `ideaId` was not found in the error message?

Reply:
I don't really know if that is needed? If it is a good practice, then I definitely should, if not, then I don't think so.